### ISSUE TITLE
You exceeded your current quota

### ISSUE THREAD

**Issue by jibanes** [JusticeRage/Gepetto#31]
For some odd reasons I get this error:
```
Request to gpt-3.5-turbo-1106 sent...
gpt-3.5-turbo-1106 could not complete the request: Error code: 429 - {'error': {'message': 'You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.', 'type': 'insufficient_quota', 'param': None, 'code': 'insufficient_quota'}}
```
I've tried to add credits to my account (by adding a credit card) but openai's billing system replied with "You are sending requests too quickly" (lol) so I couldn't even enter my credit card number, no idea what this is about, I don't have anything else behind this IP@, they must be slammed or something like that, I've noticed that there's a possibility (in config.ini: BASE_URL) to use your own model, could you please provide some examples of that with links to the model?

**Comment by JusticeRage** [comment]
Hello! I can't help with the OpenAI billing issue, and not really with setting up custom models. I know some people have done it and could use this config option, but I personally haven't. Even if I had, I think documenting the setting up of local language models would be far beyond Gepetto's scope...
If you're having trouble setting up the basic usage (gpt-3.5),I recommend not venturing into the bleeding edge stuff!

**Comment by jibanes** [comment]
Okay I got to the bottom of this, their billing api is probably less used during the night, so I tried to enter my CC# at night and this time it worked! I couldn't use Gepetto for another 30mns (same error 429) then it started to work, probably a delay between the billing provider and openai; now I'm good. I've looked into using gepetto with code llama, looks like it's workable, will give that a shot too.
